app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -28,6 +59,18 @@ def translate_git_clone(_inputs)
           '# No need for cloning, the agent takes care of that'
         end
 
+        def translate_git_tag(inputs)
+          return '# Invalid git-tag step configuration!' unless inputs.include?('tag')
+
+          cmd_all_params = "git tag -fa #{inputs['tag']} -m \"#{inputs['tag_message']}\""
+          cmd_no_message = "git tag -fa #{inputs['tag']}"
+
+          [
+            inputs.fetch('tag_message', false) ? cmd_all_params : cmd_no_message,
```

Comment:
To be consistent with the original code, this should be simplified as:

```suggestion
          [
            "git tag -fa #{inputs['tag']} -m \"#{inputs['tag_message']}\"",
```

Reply:
Switched for consistency vs the ternary implementation